### PR TITLE
[ADD] account_ux: added l10n_ar_ux report

### PR DESCRIPTION
**Pull request by feg-adhoc**

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-financial-tools/pull/541).

File: account_ux/reports/account_invoice_report_view.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                <field name="price_unit"/>
+                <field name="discount" groups="product.group_discount_per_so_line"/>
+                <field name="discount_amount" sum="Total" optional="hide" groups="product.group_discount_per_so_line"/>
+                <field name="price_subtotal_ic" sum="Total" optional="hide"/>
+            </field>
+            <field name="price_subtotal" position="attributes">
+                <attribute name="column_invisible">True</attribute>
+            </field>
+            <field name="price_total" position="after">
+                <field name="price_subtotal"/>
```
Comment (rov-adhoc) [inline account_ux/reports/account_invoice_report_view.xml]: agregar position="move"

File: account_ux/reports/account_invoice_report_view.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <attribute name="decoration-info">state == 'draft'</attribute>
+                <attribute name="decoration-muted">state == 'cancel'</attribute>
+            </xpath>
+            <field name="quantity" position="after">
+                <field name="price_unit"/>
+                <field name="discount" groups="product.group_discount_per_so_line"/>
+                <field name="discount_amount" sum="Total" optional="hide" groups="product.group_discount_per_so_line"/>
+                <field name="price_subtotal_ic" sum="Total" optional="hide"/>
+            </field>
+            <field name="price_subtotal" position="attributes">
```
Comment (rov-adhoc) [inline account_ux/reports/account_invoice_report_view.xml]: de 19 a 21 eliminar

**Comment by rov-adhoc** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@feg-adhoc @rov-adhoc linked pull request(s) ingadhoc/odoo-argentina#880 not ready. Linked PRs are not staged until all of them are ready.

**Pull request closed without merging**